packages/fx-core/src/common/template-utils/templatesActions.ts

```diff
@@ -1,7 +1,7 @@
 import AdmZip from "adm-zip";
 import fs from "fs-extra";
 import path from "path";
-import { getTemplatesFolder } from "..";
+import { getTemplatesFolder } from "../..";
```

Comment:
In today's tech discussion, Long suggest we'd better to use file path instead of "../.." to avoid cyclic reference

Reply:
Ok, let me refine it.